features/background-blend-mode.yml

```
@@ -0,0 +1,8 @@
+name: background-blend-mode
+description: The `background-blend-mode` CSS property sets how an element's background is visually mixed.
```

```suggestion
description: The `background-blend-mode` CSS property blends an element's background image and background color.
```

I've never used this so I don't know whether this is correct, but MDN seems to say it is.